docker/include/bin/start_faux

```diff
@@ -126,6 +126,12 @@ if [ -n "${options[telnet]}" ]; then
     (while true; do echo Telnet `hostname`; nc -nvlt -p 23 -e `which hostname`; done) &
 fi
 
+# SSH server is installed but not running
```

Comment:
What do you mean by "not running"?  This code explicitly starts the server... ?

Reply:
That was a note to myself which should have been removed - you're right that's exactly what it does. Removed now.